website/src/data/marketplace.js

```diff
@@ -100,21 +100,20 @@ export const extensions = [
     title: "Compose by Twitter",
```

Comment:
```suggestion
     title: "Jetpack Compose Rules",
```

Updated to the new project's name - the ruleset is no longer "by Twitter"

Reply:
Thanks! I also updated to the new project Id (`TwitterCompose` to `Compose`).